Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/manager/impl/AlternativeIdService.java

```diff
@@ -36,24 +36,19 @@ public void addAlternativeIds(String queriedId, ConceptEntry entry) {
         }
 
         if (conceptTypesService.getConceptTypeByConceptId(queriedId) == IdType.GENERIC_WORDNET_CONCEPT_ID) {
-            entry.getAlternativeIds().add(queriedId);
+            entry.setAlternativeIds(queriedId);
```

Comment:
still same as before, why are you setting the id here instead of adding it?

Reply:
It's because the alternativeIds param is a string and we are checking for duplicate while adding an id and appending it to the string. I think I need to rename the setter method.

Reply:
yes, I believe that was one of my first comments on this ;)